ui/src/components/home/NewHereSection.js

```diff
@@ -95,12 +95,15 @@ const NewHereSection = () => {
               borderWidth={'1px'}
               px={5}
               py={[10]}
+              bgColor="#F7FAFC"
+              boxShadow={'xl'}
               borderRadius={[5, 10]}
             >
               <form onSubmit={onSubmit}>
-                <VStack spacing={[4, 5]} alignItems="center">
+                <VStack spacing={[3]} alignItems="center">
                   <FormControl id="name" isRequired w={['100%']}>
                     <Input
+                      h={'3em'}
```

Comment:
![Screenshot 2022-08-20 at 11 14 12](https://user-images.githubusercontent.com/33679483/185726882-97ba244b-9d06-4189-9f63-aec9fa1b1680.png)

still seeing the same issue.

Reply:
![Screenshot 2022-08-20 at 11 15 53](https://user-images.githubusercontent.com/33679483/185726929-b98dbc1c-1078-40a2-b5fd-2becd556904a.png)
this also happen when i resize to smaller screen. i think you need to make the buttonHeights a single variable and change it accordingly